Source/AliveLibAE/BirdPortal.cpp

```diff
@@ -763,11 +763,15 @@ s32 CC BirdPortal::CreateFromSaveState_499C90(const u8* pBuffer)
 
     case PortalStates::ShrykullGetDoves_7:
     case PortalStates::Unused_8:
+    {
         if (savedState == PortalStates::Unused_8)
         {
             ALIVE_FATAL("PortalStates::Unused_8 never expected to be used");
         }
 
+        break;
```

Comment:
This probably needs to be checked against the original as case 8 previously fell through, is what what OG does?

Reply:
seems that apparently it should fall through according to mouze

Reply:
Will fix